tutorials/navigation/navigation_introduction_3d.rst

```diff
@@ -0,0 +1,152 @@
+            The region can be enabled / disabled or the use restricted with a navigationlayer bitmask.
+        - NavAgent RID
+            Reference to a specific avoidance agent with a radius value use solely in avoidance.
+
+The following SceneTree Nodes are available as helpers to work with the NavigationServer3D API.
+
+- :ref:`NavigationRegion3D<class_NavigationRegion3D>` Node
+    A Node that holds a NavigationMesh resource that defines a navigationmesh for the NavigationServer3D.
+    The region can be enabled / disabled.
+    The use in pathfinding can be further restricted through the navigationlayers bitmask.
+    Regions can join their navigationmeshes by proximity for a combined navigationmesh.
+
+-  :ref:`NavigationAgent3D<class_NavigationAgent3D>` Node
+    An optional helper Node to facilitate common NavigationServer3D API calls for pathfinding and avoidance for
+    a Node3D inheriting parent Node.
+
+-  :ref:`NavigationObstacle3D<class_NavigationObstacle3D>` Node
+    A Node that acts as an agent with avoidance radius, to work it just needs to be added under a Node3D
+    inheriting parent Node. Obstacles are intended as a last resort option for constantly moving objects
+    that cannot be re(baked) to a navigationmesh efficiently. This node also only works if RVO processing
+    is being used, more on that in the next section.
+
+The 3D navigationmeshes are defined with the following resources:
+
+- :ref:`NavigationMesh<class_NavigationMesh>` Resource
+    A resource that holds 3D navigationmesh data and provides 3D geometry baking options to define navigation
+    areas inside the Editor as well as at runtime.
+
+    - The NavigationRegion3D Node uses this resource to define its navigation area.
+    - The NavigationServer3D uses this resource to update navmesh of individual regions.
+    - The GridMap Editor uses this resource when specific navigationmeshes are defined for each gridcell.
+
+RVO processing
+--------------
+
+RVO stands for reciprocal velocity obstacle. RVO processing is a way to
+pathfind while taking into account other agents and physics bodies that
+are also moving.
+
+To use it set a target like normal. Then an agent needs to fetch its next
+nav path location, and compute its velocity to that location. Instead
+of using that value to move use it to set the velocity on the agent
+with `set_velocity`. Then a new velocity that takes into account other
+agents and obstacles is generated and emitted with the signal `velocity_computed`.
+
+However agents can only take into account a set number of other nearby
+agents, this is the :ref:`max neighbors<class_NavigationAgent3D_property_max_neighbors>`
+property of an agent and can be adjusted. This is **not** a limit for
+how many agents can use a navigation region at the same time.
+
+Setup for 3D scene
+------------------
+
+The following steps show how to setup a minimum viable RVO navigation in 3D that uses the NavigationServer3D and
```

Comment:
This is not correct as the following steps and examples do not include anything about RVO avoidance. The original was "how to setup a minimum viable navigation in 3D".